README.md

```diff
@@ -25,5 +25,17 @@ A book uploader bot that transfers documents from public libraries such as Googl
 10. Run `npm run dev` for development and `npm run start` for production.
 11. Open your browser and run your local server: http://localhost:5000
 
+## Request to Contribute
+1. Fork the repository
+2. Clone the repository to your local system by using the command : `git clone "Forked repository link` 
```

Comment:
Please change "Forked repository link" to "https://github.com/<YOUR_USERNAME>/BUB2"